lib/plugins/rotate_asg_instances/asg.rb

```diff
@@ -223,15 +252,10 @@ def wait_for_capacity
     # @param id [String] ID of the instance to terminate.
     def shutdown_instance(id)
       instance = Aws::EC2::Instance.new(id: id)
-      options = [
-        'UserKnownHostsFile=/dev/null',
-        'StrictHostKeyChecking=no'
-      ]
-      remote = "#{@ssh_user}@#{instance.public_dns_name}"
-      cmd = "'sudo shutdown -h now'"
-      remote_cmd = "ssh -o #{options.join(' -o ')} #{remote} #{cmd}"
-      SSHForkExecutor.new.run(remote_cmd)
-
+      run_ssh_command(
+        instance, 'sudo shutdown -h now',
+        ['UserKnownHostsFile=/dev/null', 'StrictHostKeyChecking=no']
```

Comment:
I think we should use the same options for all the commands here. So why dont you move these into the run ssh command method itself.

Reply:
Moved the options inside the `run_ssh_command` function.